### ISSUE TITLE
Loinc 2880-3:  "Protein [Mass/volume] in Cerebral spinal fluid"

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#3614]
Suggest creating a new term for Loinc: 2880-3 [Protein [Mass/volume] in Cerebral spinal fluid] 
New term label: Abnormal CSF protein level

New term comment (if any):
Need increased and decreased terms as well

Your biocurator ID for loinc2hpo (if desired): HPO:nvasilevsky

**Comment by pnrobinson** [comment]
see Abnormal CSF protein level and children
https://hpo.jax.org/app/browse/term/HP:0025456
